Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -259,15 +259,18 @@ void function OnNpcDeath( entity victim, entity attacker, var damageInfo )
 	
 }
 
-void function RateSpawnpoints_FD(int _0, array<entity> _1, int _2, entity _3){}
+void function RateSpawnpoints_FD( int _0, array<entity> _1, int _2, entity _3 )
+{
+
+}
 
-bool function useShieldBoost(entity player,array<string> args)
+bool function useShieldBoost( entity player, array<string> args )
 {
-	if((GetGlobalNetTime("FD_harvesterInvulTime")<Time())&&(player.GetPlayerNetInt("numHarvesterShieldBoost")>0))
+	if( ( GetGlobalNetTime( "FD_harvesterInvulTime" ) < Time() ) && ( player.GetPlayerNetInt("numHarvesterShieldBoost") > 0 ) )
```

Comment:
```suggestion
	if( ( GetGlobalNetTime( "FD_harvesterInvulTime" ) < Time() ) && ( player.GetPlayerNetInt( "numHarvesterShieldBoost" ) > 0 ) )
```
we should be consistent on the same line, either remove spaces for the first one or add spaces to the second